src/3rdparty/kissfft/test/testkiss.py

```
@@ -4,158 +4,140 @@
+        print('xver=', xver)
+        print('x2=', x2)
+        print('err', err)
         sys.exit(1)
- 
-def dofft(x,isreal):
-    dims=list( numpy.shape(x) )
-    x = flatten(x)
 
-    scale=1
-    if datatype=='int16_t':
+def dofft(x, isreal):
```

Ruff (0.11.9)

Missing return type annotation for public function `dofft`

(ANN201)